wagtail_storages/wagtail_hooks.py

```diff
@@ -1,14 +1,24 @@
 import django
 
 import wagtail
-from wagtail.documents import get_document_model
+from wagtail import VERSION as WAGTAIL_VERSION
+
+if WAGTAIL_VERSION >= (3, 0):
```

Comment:
**suggestion** using a try accept block here might save one import of VERSION

Reply:
Resolved in https://github.com/torchbox/wagtail-storages/pull/28/commits/1ab46ab175ee5f01394aab82548a7d4a3f75624d